src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
-spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=Donetsk228322!
-spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
+    spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
+    spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+    spring.datasource.username=root
+    spring.datasource.password=Donetsk228322!
+    spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
```

Comment:
what are these changes about? some formatting stuff?

Reply:
Yes I don't know why it's did like this) I'll fix it)